spec/views/hq/sponsors/_form_spec.rb

```diff
@@ -24,129 +23,95 @@ def render_sponsor_form current_sponsor
   specify 'has a form' do
     render_sponsor_form sponsor_new
 
-    assert_select 'form'
+    expect(rendered).to have_selector("form")
   end
 
   describe 'has a "Cancel" button' do
     specify 'using an existing Sponsor record' do
       render_sponsor_form sponsor_full
 
-      assert_select 'a[href=?]', hq_sponsor_path(sponsor_full.id), text: 'Cancel'
+      expect(rendered).to have_link("Cancel", hq_sponsor_path(sponsor_full.id))
     end
 
     specify 'using a new Sponsor record' do
       render_sponsor_form sponsor_new
 
-      assert_select 'a[href=?]', hq_sponsors_path, text: 'Cancel'
+      expect(rendered).to have_link("Cancel", hq_sponsors_path)
     end
   end
 
   describe 'has form values' do
     specify 'using an existing Sponsor record' do
-      allow(User).to receive(:pluck).and_return([user.user_name, user.id])
+      allow(User).to receive(:pluck).and_return([[user.user_name, user.id]])
       render_sponsor_form sponsor_full
 
       #fextfields
```

Comment:
@gmunumel : `.plunk` returns an array of arrays if you specify more than one field